django/api/tasks.py

```diff
@@ -262,3 +262,54 @@ def send_approve(recipient_email, application_id):
         cc_list=[],
         optional_subject=" – Approved",
     )
+
+
+def calculate_rebate_amount(cra_response, application_id):
+    def check_individual(primary_income):
+
+        if primary_income is None:
+            return ""
+        primary_income = int(primary_income)
+        if primary_income > 100000:
+            return ""
+        elif primary_income > 90000:
+            return "c"
+        elif primary_income > 80000:
+            return "b"
+        elif primary_income <= 80000:
+            return "a"
+
+    def check_household(primary_income, secondary_income):
+        if (primary_income is None) | (secondary_income is None):
+            return ""
+        household_income = int(primary_income) + int(secondary_income)
+        if household_income > 165000:
+            return ""
+        elif household_income > 145000:
+            return "c"
+        elif household_income > 125000:
+            return "b"
+        if household_income <= 125000:
+            return "a"
+
+    def get_final_rebate(individual_rebate, household_rebate):
+        if household_rebate == "a":
```

Comment:
I assigned 311 to you as well so we can setup something similar to a status enum for these rather than just strings.

Reply:
👍